doc/src/how_to_guide/index.rst

```diff
@@ -4,7 +4,14 @@
  How-to Guides
 ==============
 
+This section contains step-by-step guides for different developer processes.
+
+Content
+=======
+
 .. toctree::
    :maxdepth: 2
 
-   ../guides/index.rst
+   getting_started/index.rst
+   contributing/index.rst
+   guides/index.rst
```

Comment:
This creates a subsection "SymPy Guides", but it seems to me that they should just be at the top level on this page.

Reply:
How about instead of changing guides to how_to_guides/guides, we just keep guides. Then we don't need to rename what is already there.